### PR TITLE
Keep copyInfo for record fixed at starting column

### LINKED ISSUE #2109: Valid input => Formatted content is not valid F# code
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAbRAXMSBmAhgK7pYAqiAzlgPIAOWAlrAHaVh4DuA1mHALb9WYABSw0AEVhYActgB0FavSatKASjABeADoAnHSwgRgYWPIDqjLAAsAwrEGsR%2Bw8ffGReIkdgAmMABaAD5XIw8IkzMAzmsbMMjEjwA1RD0AI1hKawBPbTAAZUdEMClZBVSMrNz5GVg9fgJ0MABfdQSk4OC%2BRyEWDvdNWNsByLqAISJGdHh8rD0iRFGIgDE9An5ETnreLQ4eMAB6Q8LisB1IFmw4PUQCOjoAZnkABgvljwcWPEYAcyJ1qojHsynIsPJJtN4F8fv9AcwWPIJIh0kRfq0wpAADTgKCUbBMFi-SioMAAbVxJkpxkgABIsDk6Mg0JBGCwsNjqRA6QA3JqLUkUxIvHGJSAASRYSHZBUYAC9kKLIgAWLlgAC61JaSqpETpDKZpNZ7M5etpfPQArQQsiAEYdR5IABZAgADwAMmzEO7EETbKa7X4XmrNe5tdTQGaDcyoL74DQ8J6rgUGZgA%2B5efyYzaIn4HRmAKJShNJxVq7noPCQENah2Rx30xkxyCZWDodM081ZwXlsCPfM0yX4vRYFZsppyTjoL0d9zzRY1sN1tX6ptG1vtgeZy3Z3vKgfcgqIfiMODoVgAQSwRfjiZnB44TXxi%2BM4fc9YzjcNLI3s6gXZ3HtEgAVgfSACjoAhoEQcZEDwepEAABQIdZNiwNI-wgedEBfCA32MD9O2jddYDbP9tytclewANjAiCoJguCEPdWBODSaACHxSUeVgDigUwsBsNw1plyjNcf1IzcVwAyicw8AB2OjIOg2D4NuABVB52M4xBuN4gh%2BIfQh0GfCJQ1fUSG2IiSyK3GTd0SAAOJSGNUhC7HQTjKC%2BagFmgLB6gE4zTI8cy8Msz9rKgX87ItWTewAThclSmNuJ1j3SDCjKfHCzNrCNpKiltJPI%2BygLtEVe3A5TGLUxAHHPFggpy4T8N1KzxOikrYu7a1e1te0qvo6CLzwdC9AcQQCAEoS8qXAqxO-LrbOkuKHMDZLaoQo8TzPVhmpM3LQvy99Cs64qVrNNbyoiW1%2ByGmrRvGnbTzbfaH1m475tOxbmxi1beqoxJbX3KrJWlWgWDIPQcisf1ssO1qIqI87-quwG5PcW1QIehiLz0WAfHgZFpxPcaZoWI73DCkSFo6pbjQ5HrAL64HaKql1XXFPAyBsX0C0OgobHqLArHgeG92DOaLLpyLzrZJmAZZoG7UUjm3UlH5XXoNIDPqAtXToW5KGyd7e2AqWvpln76ebBXSuu1m7Wc9XXQAJUQG54DFiXEmVS3qZOgizoZ%2B3mfi4Gktdj2vZkIh%2BEyvQE3FdD%2BBJB9bSR2WUYZ-gSCYacrhWepGiwcayDXcP1tzSqxRj%2Bp4CdfPpi9YuGgM8mwOgGwUKg8aAH1YnFmxq2l8Ls%2B5L87ZNKubo8PxBrFTn8fWHIaD0T1qB9keH39rObbl0OZ6ViPIj8PNXZXgg143xhqDjhO0mT1P0-6-fg9%2Bo089IFui5Lju0gV0NLPJ2uZ7piivjfTeWAm4-0LogNupdO5VW7r3fyaRB6MGHqPK248D452nordGytMbGD8KDJebpkhZnGGyeAbJfjbwEo5AOxgaZtTAIRSeRUw4n2rvPHGlDXQrB8P5BEtCpQMKYbvVhEB2HI24fLY%2BxDT65nZkIsoABxbABsjZUFNiwaR5tZEaiDu1Q%2Bf1up8Lnu4Pwasl7N3geMc80BuDjHWK47AlAaAsAKBseqbZ45NQRiFQO30P62xIpdBsjsVa5hdmKScTjsBsV9EAxAyIfgsGsAiC8Up0qPz0K-RIwUqZsLMZwkOljomfliaQiAfgo5igANKIEQHQbmvNfSSj8ZsUsB1QnlPCeYghRpeEqP4e4R4tdIjOjdALE8lAbBGL9iY%2BRE9-xFTRjEjGvZHiL1mbKIkmB8b-E2OyCwKInQIDLCUlqY9ab4MUQzbZtTdmJEeBfCB05fgsBESwMRrBZS-IMgCRAZBYDg19FgAyCIBllLkRUrhmzUZWImTY4wjxwGzIvKQNILBYU8m9KwX4BTE6ZLZDktQ8L34jOedUqS6LQEeEeBQ2ZsCmClndBsdI8ACAeWqESJJ95eylNpZUz%2BNlGU7JIXswRsyJB3wIOkTACy742AKDkdkboaUPI4ci1cLy0UytUSy9RszWntKheySU7iCAAp3qK%2B5uDHkRIsVE6VbzZUfPsbM5x9ruClkoPjQmUo5DUEQI3RxXpdHGwMcUyIn0wnWzdaMqVDt3mREeAkv1KE3K3EVZQaAehGAngJeheA6lsmsGRNATy8IzZ3MRnqhRKKjU1M7HUvZTTDnzFPDAm5uqXUtFxOqJUrJKArGyKSMVLQgA)

#### Code

```fsharp
let defaultTestOptions fwk common (o: DotNet.TestOptions) =
    { o.WithCommon(
          (fun o2 ->
              { o2 with
                    Verbosity = Some DotNet.Verbosity.Normal })
          >> common
      ) with
          NoBuild = true
          Framework = fwk // Some "netcoreapp3.0"
          Configuration = DotNet.BuildConfiguration.Debug }

```
            

#### Result

```fsharp
let defaultTestOptions fwk common (o: DotNet.TestOptions) =
    { o.WithCommon(
        (fun o2 -> { o2 with Verbosity = Some DotNet.Verbosity.Normal })
        >> common
    ) with
        NoBuild = true
        Framework = fwk // Some "netcoreapp3.0"
        Configuration = DotNet.BuildConfiguration.Debug }

```
            
#### Problem description

Please describe here the Fantomas problem you encountered.
                    Check out our [Contribution Guidelines](https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#bug-reports).

#### Extra information

- [ ] The formatted result breaks by code.
- [ ] The formatted result gives compiler warnings.
- [ ] I or my company would be willing to help fix this.

#### Options

Fantomas master branch at 1/1/1990

Default Fantomas configuration

<sub>Did you know that you can ignore files when formatting from fantomas-tool or the FAKE targets by using a [.fantomasignore file](https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#ignore-files-fantomasignore)?</sub>

### PR DESCRIPTION
**Pull request by dawedawe**
Fixes #2109 
As @nojaf  suggested, it's fixed by wrapping genExpr() in atCurrentColumnIndent()

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 ### Changed
 * Update .NET SDK to 6.0.200. [#2105](https://github.com/fsprojects/fantomas/pull/2105)
 
+### Fixed
```
Comment (baronfel) [inline CHANGELOG.md:8]: Really cool to see changelogs entries as part of the PRs :)

**Review by nojaf — APPROVED** [review]
Thanks again @dawedawe!
Really wonderful to see you picking these up!

I have one nitpick about an additional blank line which I'll solve myself as it really really is a nitpick.

File: CHANGELOG.md (hunk 1 above)
Comment (nojaf) [reply CHANGELOG.md:8]: Yes, well done to have spotted this.

**Comment by nojaf** [comment]
Please try to add a more meaningful title though.
Valid input, that became invalid code is pretty much the bread and butter of Fantomas bugs.
See https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#pull-request-title

**Pull request merged**